src/plugin.h

```diff
@@ -40,7 +40,7 @@ class Plugin {
     OK,
     EVENT_CONSUMED,
     ERROR,
-  };
+  }
```

Comment:
To the best of my knowledge the declaration of an enum class without a semicolon is non standard conforming ([example](https://godbolt.org/g/S16hkY)). I did not try if this works with Arduino, but if so, this is likely to cause problems with future versions of avr-gcc and other compilers.

Reply:
Yeah, it did not work. I made the mistake of listening to cpplint, which was telling me that a class does not need a semicolon. Except this is an enum class. A subsequent patch reverted this change.

Reply:
> which was telling me that a class does not need a semicolon

Which is pretty strange because the standard dictates that a class definition must be followed by a semicolon. This is because of C++ intention to be (mostly) C compatible and there 
```cpp
struct A { } a;
```
was/is legal. The compiler must know if there is a variable definition after the class/struct. Therefore the semicolon.